notice_and_comment/static/regulations/css/less/module/header-custom.less

```diff
@@ -0,0 +1,92 @@
+/* ==========================================================================
+ reg header
+ ========================================================================== */
+
+.main-head {
+  border-bottom: 2px solid @blue;
+  height: 3.5em;
+}
+
+.sub-head {
+  top: 3.5em;
+}
+
+/*
+Application Nav
+================
+*/
+
+.app-nav {
+  a:link,
+  a:visited {
+    color: @blue;
+  }
+
+  a:hover,
+  a:active,
+  a:focus {
+    color: darken(@blue, 10%);;
+  }
+}
+
+/*
+Site Title
+================
+*/
+
+.title a:link,
```

Comment:
Can you combine the attributes like this:

`.title a {
  &:active,
  &:hover,
  &:link,
  &:visited {
  }
}`